lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -112,20 +115,20 @@ sub _option_spec {
             # employee
             'dob=s' => \$dob,
             'employeenumber=s' => \$employeenumber,
-            'end_date=s' => \$end_date,
-            'is_manager!' => \$is_manager,
+            'end-date=s' => \$end_date,
+            'is-manager!' => \$is_manager,
             'manager=s' => \$manager,
             'role=s' => \$role,
```

Comment:
How does the user know which values can be specified for the "role" argument?

Reply:
Ok. Based on the code, I figured it out. Given that you allow individual roles to be specified, don't you think this option is useless unless we have an option to query available roles?

Reply:
Added `user list a_user` to query current permissions,